src/Interfaces/hamiltonians.jl

```diff
@@ -419,3 +414,83 @@ See also [`LOStructure`](@ref Main.Hamiltonians.LOStructure).
 has_adjoint(op) = has_adjoint(LOStructure(op))
 has_adjoint(::AdjointUnknown) = false
 has_adjoint(::LOStructure) = true
+
+"""
+    AbstractOperatorColumn
+
+Abstract type for columns. See [`operator_column`](@ref).
+"""
+abstract type AbstractOperatorColumn{A,T,O} end
+
+"""
+    OffdiagonalsOperatorColumn(op::AbstractOperator{T}, address::A)
```

Comment:
```suggestion
    OffdiagonalsOperatorColumn <: AbstractOperatorColumn
```
We are not providing a constructor.